lib/factory_bot_rails/generators/null_generator.rb

```diff
@@ -1,9 +1,6 @@
 module FactoryBotRails
   module Generators
     class NullGenerator
-      def initialize(generators)
```

Comment:
This changes the arity of the initialize method, which I think means we'd fail on https://github.com/thoughtbot/factory_bot_rails/blob/57e88e79bf9d965eb4147cc111c2feff18b52c12/lib/factory_bot_rails/generator.rb#L12 if somebody has factory_bot_rails but adds configuration to disable the generators. I guess we don't have a test for disabling generators.

Reply:
ok, I can put back and add a rubocop exception. rubocop made me do it and I was dubious as well. rather not take the risk ...